Algorithms/Binary Search.py

```diff
@@ -0,0 +1,34 @@
+# BINARY SEARCH ALGORITHM
```

Comment:
Add the Question to be merged.

Reply:
i don't want to add the binary search anymore, because i realized that it was't in the hackerrank algorithms problems list. So i just removed it from the repository, and added the problem that i solved, named: "A very big number"